ext/mro/mro.xs

```diff
@@ -63,8 +63,9 @@ S_mro_get_linear_isa_c3(pTHX_ HV* stash, U32 level)
 
     /* For a better idea how the rest of this works, see the much clearer
        pure perl version in Algorithm::C3 0.01:
-       http://search.cpan.org/src/STEVAN/Algorithm-C3-0.01/lib/Algorithm/C3.pm
-       (later versions go about it differently than this code for speed reasons)
+       https://fastapi.metacpan.org/source/STEVAN/Algorithm-C3-0.01/lib/Algorithm/C3.pm
```

Comment:
Or https://metacpan.org/source/STEVAN/Algorithm-C3-0.01/lib/Algorithm/C3.pm

Reply:
Do we need to keep linking to versio 0.01, when the latest version (at present) is 0.11?

Reply:
I believe fastapi is an internal host that shouldn't be linked to directly, so the main metacpan.org link would be preferred here.